support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -1792,7 +1793,9 @@ function CheckoutComponent({
 												/>
 
 												<PayPalButton
-													env={isTestUser ? 'sandbox' : 'production'}
+													env={
+														isTestUser || !isProd() ? 'sandbox' : 'production'
```

Comment:
```suggestion
														isProd() && !isTestUser ? 'production': 'sandbox'
```
Might just be me but this seems clearer